package.json

```diff
@@ -34,15 +34,20 @@
     "url": "https://github.com/videojs/videojs-generate-postcss-config/issues"
   },
   "homepage": "https://github.com/videojs/videojs-generate-postcss-config",
+  "peerDependencies": {
+    "postcss": "^8.0.0",
+    "postcss-cli": "^8.0.0"
+  },
   "dependencies": {
-    "autoprefixer": "^9.7.4",
-    "postcss-banner": "^3.0.1",
-    "postcss-calc": "^7.0.1",
-    "postcss-csso": "^4.0.0",
-    "postcss-custom-properties": "^9.0.2",
-    "postcss-import": "^12.0.1",
+    "autoprefixer": "^10.0.1",
+    "postcss-banner": "^3.0.0",
```

Comment:
This should be downgraded?

Reply:
good catch, let me revet that. in practice it is likely no change.

Reply:
fixed